launch/axis.launch

```diff
@@ -15,7 +15,7 @@
     <node pkg="image_transport" type="republish" name="republish"
           args="compressed" if="$(arg enable_theora)">
       <remap from="in" to="image_raw" />
-      <remap from="out" to="image_raw" />
+      <remap from="out" to="image_raw_out" />
```

Comment:
Are you sure this does the right thing?

I thought the issue was the compressed topic overlapping?